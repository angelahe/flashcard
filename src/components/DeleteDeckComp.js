import React from 'react';
import '../styles/flashcard.css';

class DeleteDeckComp extends React.Component {
  constructor() {
    super();
    this.state = {
      currentDeck: '',
      deckList: [],
    };
  }

  render() {
    return (
      <div>
        <p>DeleteDeckComp</p>
      </div>
    );
  }
}
export default DeleteDeckComp;
