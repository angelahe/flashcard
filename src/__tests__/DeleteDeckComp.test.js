import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import DeleteDeckComp from '../components/DeleteDeckComp';

it('renders without crashing', () => {
  shallow(<DeleteDeckComp />);
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<DeleteDeckComp />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('snapshot of UI renders consistently', () => {
  const tree = renderer.create(<DeleteDeckComp />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('contains the container div DeleteDeckComp on shallow render', () => {
  const component = shallow(<DeleteDeckComp />);
  expect(component.html()).toContain('DeleteDeckComp');
});
