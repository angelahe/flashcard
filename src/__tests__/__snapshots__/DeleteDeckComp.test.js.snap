// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of UI renders consistently 1`] = `
<div>
  <p>
    DeleteDeckComp
  </p>
</div>
`;
